docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./app:/app
+    command: sh -c "python manage.py runserver 0.0.0.0:8000"
```

Comment:
It would help to automatically apply migrations before starting the server (after polling the DB to make sure it is reachable). Not a big deal.